src/modules/pivotTable/PivotTableEngine.js

```diff
@@ -705,7 +705,7 @@ export class PivotTableEngine {
                 dataFields.forEach((field) => {
                     const headerIndex = this.dimensionLookup.dataHeaders[field]
                     const value = parseValue(dataRow[headerIndex])
-                    if (value && !isNaN(value)) {
+                    if (!isNaN(value)) {
```

Comment:
extract this check to a fn that can be tested by a unit test? (instead of repeating it 7 times)